AdminNotificationManagerPlugin.inc.php

```diff
@@ -207,15 +226,16 @@ function _getNotificationSettingsMap() {
 	 * @return array
 	 */
 	function _getContexts() {
-		$contextDao = Application::getContextDAO();
-		$contextIterator = $contextDao->getAvailable();
-		if ($contextIterator && $contextIterator->getCount() > 1) {
-			$contextsById = array();
-			while ($context = $contextIterator->next()) {
-				$contextsById[$context->getId()] = $context->getLocalizedName();
+		$contextIdsObject = new ContextService();
+		$Ids = array();
+		if ($contextIdsObject && $contextIdsObject->getCount() >= 1) {
+			$contextsById = $contextIdsObject->getIds();
+			foreach($contextsById as $context){
+				$arrayValue = $context;
```

Comment:
For `$arrayValue`, this variable name isn't very descriptive; be sure to select variable names that tell the reader what they are.